app/controllers/phylo_trees_controller.rb

```diff
@@ -51,9 +51,8 @@ def index
                  name: taxon_lineage.name }
     end
 
-    @phylo_trees = @phylo_trees.as_json(include: { user: { only: [:id, :name] } })
-
```

Comment:
This change is unrelated to the main diff, sorry for mixing in. It completes a change I had made previously (user data is filled in a few lines below, so no need for inefficient "include: user"). I had neglected to remove the now unnecessary code.